### PR TITLE
CHE-1081: remove workspace id from services that deployed on wsagent

### PR DESCRIPTION
**Pull request by vparfonov**
Signed-off-by: Vitaly Parfonov vparfonov@codenvy.com
@skabashnyuk @sleshchenko @gazarenkov

### CODE REVIEW

**Comment by benoitf** [comment]
this is a breaking change for dashboard and I don't see any UD file modified in this PR

File: core/ide/che-core-ide-api/src/main/java/org/eclipse/che/ide/api/project/ProjectServiceClient.java [hunk 1]
```diff
@@ -33,8 +33,8 @@
     /**
      * Get all projects in current workspace.
      *
-     * @param workspaceId
-     *         id of current workspace
+     * @param devMachine
+     *         of current devMachine
```
Comment (sleshchenko) [inline core/ide/che-core-ide-api/src/main/java/org/eclipse/che/ide/api/project/ProjectServiceClient.java:37]: mb just something like `instance of current dev machine`? Or maybe we can use not current dev machine here

**Comment by sleshchenko** [comment]
It would easier to review to have separated issues for removing id from services' path and removing classes from project and maven plugins.
P.S. In issue and pull request description nothing about changes in project and maven plugins

**Comment by vparfonov** [comment]
@benoitf i talk with @ashumilova about that we will add  changes later

**Comment by vparfonov** [comment]
@skabashnyuk @garagatyi

File: wsagent/wsagent-local/src/main/java/org/eclipse/che/WorkspaceIdProvider.java [hunk 2]
```diff
@@ -8,21 +8,20 @@
...
+package org.eclipse.che;
 
-import com.google.gwt.resources.client.ClientBundle;
-
-import org.eclipse.che.ide.ui.Styles;
 
 /**
- * @author Evgen Vidolob
+ * //
+ *
```
Comment (skabashnyuk) [inline wsagent/wsagent-local/src/main/java/org/eclipse/che/WorkspaceIdProvider.java:18]: javadoc

**Comment by skabashnyuk** [comment]
other ok

**Comment by garagatyi** [comment]
LGTM

**Comment by gazarenkov** [comment]
OK for me

**Pull request merged**